website/docs/r/app_service_managed_certificate.html.markdown

```diff
@@ -0,0 +1,114 @@
+---
+subcategory: "App Service (Web Apps)"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_app_service_managed_certificate"
+description: |-
+  Manages an App Service Managed Certificate.
+---
+
+# azurerm_app_service_managed_certificate
+
+Manages a App Service Managed Certificate.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "exampleRG"
+  location = "West Europe"
+}
+
+resource "azurerm_app_service_plan" "example" {
+  name                = "examplePlan"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  kind                = "Linux"
+  reserved            = true
+
+  sku {
+    tier = "Basic"
+    size = "B1"
+  }
+}
+
+resource "azurerm_app_service" "example" {
+  name                = "exampleApp"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+  app_service_plan_id = azurerm_app_service_plan.example.id
+}
+
+resource "azurerm_dns_txt_record" "test" {
+  name                = join(".", ["asuid", "%s"])
```

Comment:
I presume this wants to be:

```suggestion
  name                = "mycustomhost.contoso.com"
```

Reply:
This needs to be created as the hostname to be used, prefixed with `asuid.`